RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/Program.cs

```diff
@@ -0,0 +1,292 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using RefrigeratorSaraFarkash;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+
+namespace exeRefreg
+{
+    class Program
+    {
+        public static List<CRefrigerator> SortRefrigerator(List<CRefrigerator> refrigerators)
+        {
+            refrigerators.Sort();
+            return refrigerators;
+        }
+        public static void inputKoshroot(CItem cItem)
+        {
+            Console.WriteLine("please enter Koshroot");
+            try
+            {
+                cItem.Kashroot = Console.ReadLine();
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+                inputKoshroot(cItem);
+            }
+        }
+
+        public static void inputName(CItem cItem)
+        {
+            Console.WriteLine("please enter Name");
+            try
+            {
+                cItem.Name = Console.ReadLine();
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+                inputName(cItem);
+            }
+        }
+        public static void inputTakeSpace(CItem cItem)
+        {
+            Console.WriteLine("please enter TakeSpace");
+            try
+            {
+                cItem.TakeSpace = double.Parse(Console.ReadLine());
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+                inputTakeSpace(cItem);
+            }
+        }
+
+        public static void inputTypeI(CItem cItem)
+        {
+            Console.WriteLine("please enter TypeI");
+            try
+            {
+                cItem.TypeI = Console.ReadLine();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+                inputTypeI(cItem);
+            }
+        }
+
+        public static void inputExpiryDate(CItem cItem)
+        {
+            Console.WriteLine("please enter TypeI");
+            try
+            {
+                cItem.ExpiryDate = DateTime.Parse(Console.ReadLine());
```

Comment:
you should tell the user the correct format to enter the date

Reply:
fixed